packages/amplify-graphql-auth-transformer/API.md

```diff
@@ -117,6 +117,8 @@ export class AuthTransformer extends TransformerAuthBase implements TransformerA
     // (undocumented)
     addFieldsToObject: (ctx: TransformerTransformSchemaStepContextProvider, modelName: string, ownerFields: Array<string>) => void;
     // (undocumented)
+    addIamAuthToCustomTypes: (ctx: TransformerTransformSchemaStepContextProvider) => void;
+    // (undocumented)
```

Comment:
It seems this could be private.
(on the other hand, we never paid a lot of attention to transformers public apis...).

Reply:
Yeah. I haven't formed a strong opinion here yet. I'm just following the established pattern for now. 🤷

Reply:
I would recommend adopting "if something can be private it should be". that creates less liability going forward. (we've been doing this for a while on backend side).
For transformers it would be more to form a habit (or maybe stop the bleeding) rather than fix them. Their apis are already overexposed.

Reply:
> Their apis are already overexposed

This might be a sign that all these transformers packages could have been directories in single package.

Reply:
I might to agree. And, it all seems pretty chaotic to me at the moment. But, there's also a ton of *intrinsic* complexity in the problem space. So, I'm attempting to respect what came before and keep as many existing fences erected until I understand them better and have more solid opinions.